src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -325,17 +325,34 @@ public int getSubTrackCount() {
 	}
 
 	public boolean isVideo() {
-		return videoTrackCount > 0;
+		return getMediaType().equals(MediaType.VIDEO);
 	}
 
 	public boolean isAudio() {
-		return videoTrackCount == 0 && audioTracks.size() == 1;
+		return getMediaType().equals(MediaType.AUDIO);
 	}
 
 	public boolean hasAudio() {
 		return audioTracks.size() > 0;
 	}
 
+	public MediaType getMediaType() {
+		if (videoTrackCount > 0) {
```

Comment:
I'm aware of that, do you have a better definition? What I don't get is why somebody would mux still images as video - is that really a wide spread "problem"?